### ISSUE TITLE
Hekili not recommending use of trinket "Tablet of Despair" after receiving a higher level version of the item.

### ISSUE THREAD

**Issue by Fraxien64** [Hekili/hekili#1079]
- I am on the latest version (as far as I know) - v9.0.5-1.0.7

- I do not see any posts about this

I am on a Warlock. When I was using a lower level version of the trinket "Tablet of Despair", Hekili would correctly recommend me to use the trinket. After receiving a higher level version of the item, it no longer shows up on the queue.

This issue has been present for a couple days now, starting only upon receiving the upgraded version of the trinket.

I have tried:
- Changing what the trinket is set as (default, cooldowns, defensives, etc), still does not recommend using the trinket.
- Changing the keybind the trinket is on
- Changing specialization
- Disabling and enabling the addon
- Reinstalling the addon
- Unequipping and reequipping the trinket
- Temporarily removing other trinkets so it is the only one present

Character Data pastebin: https://pastebin.com/E99rXKSp

**Comment by Hekili** [comment]
A snapshot would be helpful for diagnosing this.

2.  If reporting a disagreement with the addon's recommendations, please take a snapshot when you observe the issue and provide the snapshot.  This is not a screenshot.  A snapshot can be generated by binding the Pause feature on the Toggles tab.  When you press Pause, a snapshot of how the addon decided on its CURRENT recommendations is generated and saved to the Snapshots tab.  Copy and paste this to pastebin.com and provide the link here.
 
**PUT SNAPSHOT LINK HERE:**

**Comment by Fraxien64** [comment]
I'll put in two snapshots, one with my second trinket on cooldown and one with if off cooldown.

With both trinkets available to use: https://pastebin.com/iUNAxe3a

With Soul Igniter trinket on cooldown and with Tablet of Despair available for use: https://pastebin.com/F4Eq37uq

(let me know if I did the snapshots incorrectly, first time using them)

Side note: When I was checking to make sure I had both trinkets under the correct category (Default, cooldowns), I noticed the name for Soul Igniter was purple, and the name for Tablet of Despair is green. My Tablet of Despair is purple in my inventory, so maybe it just doesn't recognize the purple-tier version of the Trinket? dunno I ain't a dev

Screenshot in case what I said was unclear lol: https://prnt.sc/11y4ole
